src/lib/components/documents/NoDocuments.svelte

```diff
@@ -0,0 +1,59 @@
+<script>
+  import { _ } from "svelte-i18n";
+
+  // SVG assets
+  import emptyResultsSvg from "@/assets/empty_results.svg?raw";
+</script>
+
+<div class="no-results">
+  <h2>{$_("noDocuments.noSearchResults")}</h2>
+  <div class="img">
+    {@html emptyResultsSvg}
+  </div>
```

Comment:
Just trying to move towards unified icon set:

```suggestion
  // SVG assets
  import { Search24 } from "svelte-octicons";
</script>

<div class="no-results">
  <h2>{$_("noDocuments.noSearchResults")}</h2>
  <div class="img">
    <Search24 height="32" width="32" />
  </div>
```

I have an "empty" common component in my current branch that I'll plop in here once its merged in.